org.lflang/src/org/lflang/cli/Lfc.java

```diff
@@ -229,7 +229,7 @@ private Path getActualOutputPath(Path root, Path path) {
      *
      * @return Properties for the code generator.
      */
-    protected Properties filterPassOnProps() {
+    public Properties cliArgsToBuildParams() {
```

Comment:
With this name I would expect the return type to be `List<BuildParm>` or something along those lines. Maybe it should just be called `getCliArgs`?

Reply:
It doesn't return all the CLI args either—just those required by the generator—so I called it `getGeneratorArgs` instead.